tests/checker/test_http.py

```diff
@@ -60,7 +60,7 @@ def _test_status(self, status):
             resultlines.append("warning No Content")
         if status == 429:
             resultlines.append("warning Rate limited (Retry-After: None)")
-        if (status not in [101, 102] and status < 200) or (
+        if (status not in [101, 102, 103] and status < 200) or (
```

Comment:
How is this related to dnspython API deprecation, and what is HTTP status code 103?

Reply:
Once again the git commit message answers my question!

I wish github made it more natural to do reviews commit-by-commit rather than in one lump.

Reply:
i would rather the code answer the question than having to dig into git history. both places is best.